source/megengine-schema.js

```diff
@@ -0,0 +1,2575 @@
+var $root = flatbuffers.get('megengine');
```

Comment:
done. Since it takes some time to put fbs in the official repo, the links in sync are personal, and will be modified after fbs is put in the official repo

Reply:
Why are the `.fbs` files not in the official repo? Is this a prototype?

Reply:
Because there was no need to trigger us to put these files into the official repo before, and now we are going through the open source process, which may take some time to synchronize these files.

Not exactly a prototype. In order to adapt tools/flatc.js, I have made some modifications, such as adding a complete namespace prefix and disabling union array.

Reply:
done